assets/olive-optimizer/general/jobs/olive-optimizer-built-in-component/pipeline.yml

```diff
@@ -0,0 +1,20 @@
+$schema: https://azuremlschemas.azureedge.net/latest/pipelineJob.schema.json
+type: pipeline
+display_name: olive-optimizer-built-in-component
```

Comment:
`name` should be used instead of `display_name`. If you want to have a more readable display name, add one as well. See example [here](https://github.com/Azure/azureml-assets/blob/71102c510ab4006ba146636ef4f6f2e58d673af3/assets/large_language_models/components_pipelines/data_ingestion_git_to_acs_basic/spec.yaml#L8-L9).